guides/common/modules/proc_configuring-an-external-dhcp-server.adoc

```diff
@@ -32,16 +32,9 @@ include::snip_firewalld.adoc[]
 +
 [options="nowrap"]
 ----
-# dnssec-keygen -a HMAC-MD5 -b 512 -n HOST omapi_key
-----
-+
-As a result, a key pair that consists of two files is created in the current directory.
-. Copy the secret hash from the key:
-+
-[options="nowrap"]
-----
-# grep ^Key Komapi_key.+*.private | cut -d ' ' -f2
+# tsig-keygen -a hmac-md5 omapi_key
```

Comment:
"omapi_key" would normally be a replaceable but it looks like Foreman expects the key to be named exactly this..? Do you think it would be worth letting users know? I'm thinking that perhaps you could update the step lead-in sentence to something like "Generate a security key named omapi_key:".

Reply:
The name *can* be changed, but needs to be configured in other places (look for `--foreman-proxy-plugin-dhcp-remote-isc-key-name`).

Reply:
Perhaps a replaceable named _My_Omapi_Key_Name_ then, used in both procedures? _My_Something_ seems to be the standard throughout the docs set.

It's especially confusing in the step to modify `/etc/foreman-proxy/settings.d/dhcp.yml` in the other procedure where some values are labeled as replaceables (_My_Something_) but omapi_key isn't. That's actually what made me think this was the only permitted name for the key.